src/main/solr/dssolr/conf/schema.xml

```diff
@@ -435,6 +440,23 @@ In solrconfig.xml the classic schema is defined with this:  <schemaFactory class
     <?example     Kagerester?>
   </field>
 
+  <field name="broadcaster" type="string">
```

Comment:
As `string` does not help with search (no normalisation and such), we should have a `copyField` from `broadcaster` to `creator_affiliation_generic`.

Reply:
I see your point, however the values in `broadcaster` and `creator_affiliation_generic` are different .`broadcaster` is an official organisation name like 'DR' or 'TV 2 Danmark'. While  `creator_affiliation_generic` is a shorthand for a channel like 'drp1'

Reply:
Okay, then this will be part of the overarching issue of how we handle fields in Solr. Postponed for later.